src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
+* {
+  margin: 0;
```

Comment:
margin: 0, better stay only in body selector

Reply:
Done